README.md

```diff
@@ -24,9 +26,30 @@ You may of course as well go ahead and download the generated files [here](https
 3. Power on the device, wait about a minute for it to be ready.
 4. Open a web browser and navigate to http://192.168.1.1 and wait for the wizard to come up.
 5. Click *exactly* inside the radio button to confirm you have read the terms and conditions, then abort the wizard.
-6. At the login screen you are then being thrown at password 'admin' gets you back in.
+6. You should then be greeted by the login screen, the stock password is "admin". (This step might only apply to the Linksys, everything else remains the same) 
 7. Navigate to __Administration__ -> __Firmware Upgrade__
 8. Upload `openwrt-mediatek-mt7622-linksys_e8450-ubi-initramfs-recovery-installer.itb` to vendor web interface upgrade page.
 9. Wait for OpenWrt recovery image to come up.
-10. Login and navigate to _System_ -> _Backup / Flash Firmware_ and then upload `openwrt-mediatek-mt7622-linksys_e8450-ubi-sysupgrade.itb` there.
-11. Go ahead and setup OpenWrt
+10. Login and navigate to _System_ -> _Backup / Flash Firmware_ (Consider taking another backup of every MTDblock category before proceeding)
+11. Upload the `openwrt-mediatek-mt7622-linksys_e8450-ubi-sysupgrade.itb` file.
+12. Reboot and proceed to a normal OpenWRT setup (or upload your configuration file).
+
+## Convert "release v0.3" to OpenWRT snapshot release.
+**WARNING** SNAPSHOT RELEASES ARE LARGELY UNTESTED! 
+THIS WILL ALSO REMOVE LUCI PACKAGES RESULTING IN A "HEADLESS" ROUTER WITH STOCK SETTINGS!
+PROCEED AT YOUR OWN RISK!
+1. If you haven't already, backup every "mtdblock" category, and move the "boot_backup" to another device. (In the event of emergency you can reflash via [JTAG](https://openwrt.org/toh/linksys/linksys_e8450)
+2. Backup your configuration file which can be loaded after.
+3. Upload the latest UBI [sysupgrade.itb](https://downloads.openwrt.org/snapshots/targets/mediatek/mt7622/openwrt-mediatek-mt7622-linksys_e8450-ubi-squashfs-sysupgrade.itb)
```

Comment:
I'd recommend people to install `auc`, in that way all configuration and package selection is preserved.
```
opkg install auc
auc
```